examples/django/app/types.py

```diff
@@ -43,18 +43,14 @@ class ColorOrder:
 # types
 
 
-@strawberry_django.type(
-    models.Fruit, filters=FruitFilter, order=FruitOrder, pagination=True
-)
+@strawberry_django.type(models.Fruit, filters=FruitFilter, order=FruitOrder, pagination=True)
```

Comment:
This project has a black configuration of 80 max characters. Your black formatting probably did not use that config and reformatted it here

Reply:
I was just copying from SD+ thought it would be helpful, I can delete that...
where is the 80 lines configured?